core/src/zeit/content/audio/browser/audioupdate.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import zope.component
+
+import zeit.simplecast.interfaces
+import zeit.cms.browser.menu
+from zeit.cms.i18n import MessageFactory as _
+
+
+log = logging.getLogger(__name__)
+
+
+class AudioUpdate(zeit.cms.browser.view.Base):
+    def __call__(self):
+        audio = self.context
+        simplecast = zope.component.getUtility(zeit.simplecast.interfaces.ISimplecast)
+        if not audio.external_id:
+            return
+        episode_data = simplecast._fetch_episode(audio.external_id)
+        if episode_data:
+            simplecast._update(audio, episode_data)
+            message = _('successfully updated.')
+            message = f'{audio.uniqueId} {message}'
```

Comment:
Die Übersetzung der beiden Nachrichten funktioniert nicht, da du das Argument in die Übersetzung mit aufnehmen musst :)

```suggestion
            message = _('${name} successfully updated.', mapping={'name': audio.uniqueId},)
```